### PR TITLE
part8_commit

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Splunkbase Download App:
 [https://splunkbase.splunk.com/app/5351/](https://splunkbase.splunk.com/app/5351/)
 
 ## Overview
-The Cyences App for Splunk was designed to allow users complete visibility of their system's security status. It was created with the intention of becoming a seamless piece of the Blue team arsenal for security engineers and administrators. The forensic interface based on the MITRE ATT&CK framework to quickly identify areas of concern. continuous quarterly enhancements are occurring towards achieving that goal. These updates will continue to improve tool customization and scalability with advanced search features, alerting, machine learning, and AI.
+The Cyences App for Splunk was designed to allow users complete visibility of their system's security status. It was created with the intention of becoming a seamless piece of the Blue team arsenal for security engineers and administrators; continuous quarterly enhancements are occurring towards achieving that goal. These updates will continue to improve tool customization and scalability with advanced search features, alerting, machine learning, and AI.
```
Comment (VatsalJagani) [inline docs/index.md:17]: The forensic interface based on the MITRE ATT&CK framework to quickly identify areas of concern.
- We need to keep this line.

File: docs/user_guide/alerts_dashboards.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ The Cyences app utilizes the data provided by the aforementioned vendor products
 
 The Vulnerability dashboard is designed to view the vulnerability count by severity, new vulnerabilities found over time, a vulnerability summary based on host(s), and a list of vulnerabilities. Splunk users can further use the drilldown capability to view a vulnerability list for a single host or a list of hosts affected by a vulnerability.
 
-Dashboard panels:
```
Comment (VatsalJagani) [inline docs/user_guide/alerts_dashboards.md:326]: dashboard panels is more accurate actually.

File: docs/index.md (hunk 1 above)
Comment (mahirchavda) [reply docs/index.md:17]: resolved in my PR.

File: docs/user_guide/alerts_dashboards.md (hunk 2 above)
Comment (mahirchavda) [reply docs/user_guide/alerts_dashboards.md:326]: resolved in my PR.

**Pull request merged**